### DISCUSSION TITLE
LSW3_15_1216_1.06 / 234XXXXXXX

### DISCUSSION THREAD

**Discussion by chimj** [StephanJoubert/home_assistant_solarman#134] [Q&A]
Update, 

I got a few steps further along the way...

Found a lot of input registers 

input,3001,121
input,3036,58
input,3038,20
input,3040,20
...

and holding registers
holding,3200,8258
holding,3201,27765
...

Installed via Hacs, my solarman_status_connection sensor now states as connected.
I tried a few registers but values still appear as 'Unknown'.

I also noticed most configuration files use hex registers versus decimal registers, can home_assistant_solarman take both ?
Or how do I go about debugging this further, the log files don't state anything.

requests:
  - start: 0x3200
    end: 0x3617
    mb_functioncode: 0x04
  - start: 0x3200
    end: 0x3228
    mb_functioncode: 0x03
    
parameters:
 - group: Battery
   items:
    - name: "PV_Generation_Pwr"
      class: "power"
      state_class: "measurement"
      uom: "W"
      scale: 0.1
      rule: 5
      registers: [ 0x3036 ]
      icon: 'mdi:solar-power'          
    - name: "PV_Generation_Pwr_Total"
      class: "power"
      state_class: "measurement"
      uom: "W"
      scale: 0.1
      rule: 1
      registers: [ 0x3036]
      icon: 'mdi:solar-power'   

Thank you.

**Comment by pibi73** [comment]
Hi,
do you have any way to use LSW3_15_1216_1.06 with home assistant?

thanks
